cfgov/v1/tests/atomic_elements/organisms/test_organisms.py

```diff
@@ -183,7 +184,27 @@ def test_tableblock(self):
         self.assertContains(response, 'Header One')
         self.assertContains(response, 'Row 1-1')
         self.assertContains(response, 'Row 2-1')
-        
+
+    def test_tableblock_missing_attributes(self):
+        """Table correctly displays when value dictionary is missing attributes"""
+        value = atomic.table_block.get( 'value', None )
+        del value['first_row_is_table_header']
+        del value['first_col_is_header']
+        table = TableBlock()
+        html = str(table.render(table.to_python(value)))
+
+        self.assertRegexpMatches(html, 'Header One')
+        self.assertRegexpMatches(html, 'Row 1-1')
+        self.assertRegexpMatches(html, 'Row 2-1')
+
+        self.assertEqual(value.get('first_row_is_table_header', None) , None)
```

Comment:
Simpler alternative: `self.assertIsNone(value.get('first_row_is_table_header'))`. Default value of `dict.get` is `None` if the key doesn't exist.